<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenuBar {
background: rgb(9, 9, 9);
color: rgb(255, 255, 255);
}
QMainWindow {
background: rgb(64, 32, 64);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gWidget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>320</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>85</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(38, 38, 38);
color: rgb(234, 234, 234);
}</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stepBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>85</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(38, 38, 38);
color: rgb(234, 234, 234);
}

QPushButton::disabled {
color: rgb(166, 166, 166);
background-color: rgb(0, 0, 0)
}</string>
    </property>
    <property name="text">
     <string>step</string>
    </property>
   </widget>
   <widget class="QListWidget" name="memView">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>240</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget{
background-color: rgb(0, 0, 0);
color: rgb(217, 217, 217)
}

QListWidget::item::selected {
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 0);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="pcLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>PC:</string>
    </property>
   </widget>
   <widget class="QLabel" name="opLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>OP:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: none;
border: none;
text-align: left;
font-size: 12px;
color: rgb(255, 255, 255);
}
QPushButton::hover {
text-decoration: underline;
font: bold;
}</string>
    </property>
    <property name="text">
     <string>more</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="OpenFile"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="kbdSetting"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetting"/>
  </widget>
  <action name="OpenFile">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>open game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="kbdSetting">
   <property name="text">
    <string>Keyboard Setting</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
